config/passport.js

```diff
@@ -8,20 +8,20 @@ passport.use(
   new LocalStrategy(
     {
       usernameField: 'user[email]',
-      passwordField: 'user[password]',
+      passwordField: 'user[password]'
     },
-    ((email, password, done) => {
+    (email, password, done) => {
```

Comment:
@chingsley any particular reason why you're making lots of formatting changes?

Reply:
This code is from the 'registration branch'. When I resolved a merge conflict after pushing to staging, I 'inherited' inherits from other branches such as the this one from (feat-registration-#164270478) by @leksyib and from ft-view-profile-page-#164269844 by @olajide1234. I really don't know how that happened.